frontend/src/plugin-loader.tsx

```diff
@@ -294,12 +321,14 @@ class PluginLoader extends Logger {
     includeFiles?: boolean,
     regex?: RegExp,
   ): Promise<{ path: string; realpath: string }> {
+    const { t } = useTranslation();
+
     return new Promise((resolve, reject) => {
       const Content = ({ closeModal }: { closeModal?: () => void }) => (
         // Purposely outside of the FilePicker component as lazy-loaded ModalRoots don't focus correctly
         <ModalRoot
           onCancel={() => {
-            reject('User canceled');
+            reject(t('PluginLoader.file_picker_cancel_text'));
             closeModal?.();
```

Comment:
APIs like this that are passed to plugins should never be localized! This can cause issues in plugins and is never displayed to users!

Reply:
Reverted now.